vit-servicing-station-lib/src/v0/endpoints/proposals/requests.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ProposalVoteplanIdAndIndex {
+    pub voteplan_id: String,
+    pub indexes: Vec<i64>,
```

Comment:
why signed? Although I guess it's something related to the db.

Reply:
Yep, db related (sadly).